include/pqxx/internal/conversions.hxx

```diff
@@ -1156,6 +1156,37 @@ inline constexpr format param_format(std::vector<std::byte, Args...> const &)
 template<typename T> inline constexpr bool is_sql_array<std::vector<T>>{true};
 
 
+#if defined(PQXX_HAVE_SPAN) && __has_include(<span>)
```

Comment:
I think one thing is missing: there's no `pqxx::type_name` template.

Reply:
what should it be? I'm seeing that vector is using default implementation, span could also, but i could add something like:
```cpp
template<class T, size_t Extent> inline std::string_view const type_name<std::span<T, Extent>>{"array"};
template<size_t Extent> inline std::string_view const type_name<std::span<std::byte, Extent>>{"binarystring"};
```

Reply:
On second thought...  On the most popular compilers I think the type name will get a decent automatic definition.  So never mind!